src/tasks/tests/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks.Tests.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
+    <Nullable>enable</Nullable>
+    <IsTestProject>true</IsTestProject>
```

Comment:
This test project is being included in source-build when it shouldn't.  This is where all of the prebuilts are coming from.  @ViktorHofer can you advise on the appropriate constructs to use here to exclude?  I would have expected IsTestProject to be sufficient.

Reply:
Tests need to be excluded here: https://github.com/dotnet/runtime/blob/85078215aa7039da526c993cc8d4f5deaae6cbe9/src/tasks/tasks.proj#L9-L10

Ideally via a glob pattern.